plugins/donum-paper/src/main/java/com/github/civcraft/donum/gui/DeathInventoryGUI.java

```diff
@@ -133,7 +133,7 @@ public void clicked(Player arg0) {
                     @Override
                     public void clicked(Player p) {
                         Donum.getManager().returnDeathInventory(i);
-                        Donum.getInstance().info(p.getName() + " returned inventory " + i.getID());
+                        Donum.getInstance().info(p.getName() + " returned inventory " + i.getID() + " to delivery inventory of " + NameAPI.getCurrentName(viewer));
```

Comment:
only thing to really further enhance this would be to include the date the inventory was returned. e.g returned inventory blah to delivery inventory of Diet_Cola at Nov 3, 13:00 etc

Reply:
Doesn't the logger include timestamps by default?

Reply:
I think thru logs yes, when I was admin we always had fuckery with the gui as in when was something returned and to ensure we werent double returning items. Maybe something for a different pr tho

Reply:
I was going to create a gui to see, remove, and add items, but I didn't feel like tracking all that state, so I didn't.